database/index.js

```diff
@@ -12,9 +12,33 @@ db.once('open', () => {
 })
 
 const reviewSchema = new mongoose.Schema({
-
+  versionKey: false,
+  campId: {
+    unique: true,
+    type: Number,
+    sparse: true
```

Comment:
this is interesting. I don't really know what it does. lol I'll look into it

Reply:
I'm not 100% sure either.